docs/questionsAPI.md

```diff
@@ -6,34 +6,32 @@ Gets the list of default questions as configured
 
 ```json
 {
-  "info": {
```

Comment:
Should also be updated in docstring in `main.py`. Possibly we should remove these .md files and have them be auto-generated from docstrings? Or just look at docstrings when we need to?

Reply:
docstrings are better yes

Reply:
but this change is to bring the API doc in line with the python, as you didn't return the reply with an "info" element.
Info was used because the other part was "blacklist", but we need neither as blacklist will now be a flag in the appropriate part of the response, not a list of blacklisted IDs as per the old PHP API